SH4RE/SH4RE/CreateListing/CreateListingView.swift

```diff
@@ -555,3 +555,17 @@ struct CreateListingView: View {
         .ignoresSafeArea(.keyboard, edges: .bottom)
     }
 }
+
```

Comment:
I would like to enable this preview but this file has errors that need to be resolved first